sphinx_book_theme/layout.html

```diff
@@ -20,7 +20,25 @@
 <main class="col py-md-3 pl-md-4 bd-content overflow-auto" role="main">
     {% block docs_body %}
     {% include "topbar.html" %}
-    <div id="main-content" class="row">
+    <h1 class="onlyprint">{{ pagetitle }}</h1>
```

Comment:
Why don't we just wrap all of this in a div like `<div id="jb-print-wrapper" class="onlyprint">`? I feel like that'd be the most obvious to understand and then we only need one `onlyprint` in there

Reply:
If this were the case, could we just put the TOC code:

https://github.com/executablebooks/sphinx-book-theme/blob/master/sphinx_book_theme/topbar.html#L21-L34

in a standalone `html` file, and then import it there and here, so we don't repeat ourselves?

Reply:
@choldgraf there is a bit of difference in classes between the toc in print and non-print.